dockers/docker-orchagent/orchagent.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# Export platform information. Required to be able to write
+# vendor specific code.
+export platform=`sonic-cfggen -m /etc/sonic/minigraph.xml -v onie_switch_asic`
```

Comment:
Definition of platform:
https://github.com/Azure/sonic-swss/blob/27c49a0bf3a71556b553aa30fe2a7c1cdb996142/orchagent/copporch.cpp#L16
https://github.com/Azure/sonic-swss/blob/476772db125f3791a986d8c7d6f89b6d812e6234/orchagent/routeorch.cpp#L17
Usage:
https://github.com/Azure/sonic-swss/blob/27c49a0bf3a71556b553aa30fe2a7c1cdb996142/orchagent/copporch.cpp#L117
https://github.com/Azure/sonic-swss/blob/476772db125f3791a986d8c7d6f89b6d812e6234/orchagent/routeorch.cpp#L51